docs/README.md

```diff
@@ -110,4 +110,4 @@ The `Source\Example Measurements` directory contains example measurement service
 
 ## Creating a LabVIEW executable for static registration
 
-A build specification is included with the generated measurement service. To create an executable for the measurement service, right-click the build specification and select `Build`.
+Two build specifications are included with the generated measurement service. The UI must be built before building the measurement service executable build specification is built. To create an executable for the measurement service, right-click the build specification and select `Build`.
```

Comment:
I think this new second sentence here has an extraneous verb / verb clause.  Recommend clean that up in place.  Or, an option is to replace the whole block with something like this copilot-assisted content:

> The generated measurement service includes two build specifications: one for the UI and another for the service executable. The UI build specification must be constructed prior to the measurement service executable. To construct these build specifications, right-click on the desired build specification and choose 'Build'.